bugsnag-android-core/src/main/java/com/bugsnag/android/internal/ImmutableConfig.kt

```diff
@@ -211,6 +232,8 @@ internal fun sanitiseConfiguration(
         } else {
             configuration.logger = NoopLogger
         }
+    } else {
+        validateApiKey(configuration.apiKey)
     }
```

Comment:
This shouldn't be an in `else` since as things stand, we are only validating the API-Key when we don't configure the logger